index.js

```diff
@@ -16,15 +30,28 @@ const parseFinalResult = (result, event) => {
 
   // Old writes accepted a list, but CLI doesn't anymore, which matters for _write and _post_write
   if (event.name === 'create.write' || event.name === 'create.post') {
-    if (_.isArray(result) && result.length) {
+    if (Array.isArray(result) && result.length) {
       return result[0];
-    } else if (!_.isArray(result)) {
+    } else if (!Array.isArray(result)) {
       return result;
     }
 
     return {};
   }
 
+  if (
+    event.name.endsWith('.input') ||
+    event.name.endsWith('.output') ||
+    event.name.endsWith('.input.post') ||
+    event.name.endsWith('.output.post')
+  ) {
+    if (Array.isArray(result)) {
+      result.forEach(field => {
+        field.type = FIELD_TYPE_CONVERT_MAP[field.type] || field.type;
+      });
+    }
+  }
```

Comment:
Only deals with field type conversion for now. We'll need to do more to convert custom fields from WB to CLI.